WooCommerce/WooCommerceTests/Extensions/StringHelpersTests.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+@testable import WooCommerce
+
+/// String+Helpers: Unit Tests
+///
+class StringHelpersTests: XCTestCase {
```

Comment:
nit: we usually mark a class as `final` if we don't see it being subclassed in the near future (which is most likely the case for test classes)

```suggestion
final class StringHelpersTests: XCTestCase {
```